src/libertem/web/cli.py

```diff
@@ -13,16 +14,19 @@
               help='enable/disable opening the browser', default='True')
 @click.option('-l', '--log-level', help=f"set logging level. Default is 'info'. {log_values}",
               default='INFO')
+@click.option('--instance_type', type=click.Choice(["liberated", "restricted"]), default="liberated")
```

Comment:
This allows users to choose what kind of session they want to run the libertem server in. I have setup two. But I believe it would be better to add more "modes" that fill in the gaps for special use cases.